src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    or replica sets within the common infrastructure.
+    - The values must be used consistently, not changed after the initial setup.
+    The initial values are stored in snapshot files and are checked whenever
+    the system is restarted.
+    - The values must comply with RFC 4122. The nil UUID is not allowed.
+]])
+
+M['database.mode'] = format_text([[
+    An instance's operating mode. This option is in effect if
+    `replication.failover` is set to `off`.
+
+    The following modes are available:
+
+    - `rw`: an instance is in read-write mode.
+    - `ro`: an instance is in read-only mode.
+
+    If not specified explicitly, the default value depends on the number of
+    instances in a replica set. For a single instance, the `rw` mode is used,
+    while for multiple instances, the `ro` mode is used.
+]])
+
+M['database.replicaset_uuid'] = format_text([[
+    A replica set UUID.
+
+    By default, replica set UUIDs are generated automatically.
+    `database.replicaset_uuid` can be used to specify a replica set identifier
+    manually.
+]])
+
+M['database.txn_isolation'] = 'A transaction isolation level.'
+
+M['database.txn_timeout'] = format_text([[
+    A timeout (in seconds) after which the transaction is rolled back.
+]])
+
+M['database.use_mvcc_engine'] = 'Whether the transactional manager is enabled.'
+
+-- }}} database configuration
+
+-- {{{ failover configuration
+
+M['failover'] = format_text([[
+    The `failover` section defines parameters related to a supervised failover.
+]])
+
+M['failover.call_timeout'] = format_text([[
+    A call timeout (in seconds) for monitoring and failover
+    requests to an instance.
+]])
+
+M['failover.connect_timeout'] = format_text([[
+    A connection timeout (in seconds) for monitoring and failover
+    requests to an instance.
+]])
+
+M['failover.lease_interval'] = format_text([[
+    A time interval (in seconds) that specifies how long an instance should
+    be a leader without renew requests from a coordinator. When this interval
+    expires, the leader switches to read-only mode. This action is performed
+    by the instance itself and works even if there is no connectivity between
+    the instance and the coordinator.
+]])
+
+M['failover.probe_interval'] = format_text([[
+    A time interval (in seconds) that specifies how often a monitoring service
+    of the failover coordinator polls an instance for its status.
+]])
+
+M['failover.renew_interval'] = format_text([[
+    A time interval (in seconds) that specifies how often a failover coordinator
+    sends read-write deadline renewals.
+]])
+
+M['failover.replicasets'] = format_text([[
+    Configurate failover priorities for each replicaset. It maps replicaset
+    names to instance priorities, where lower priority values are preferred
+    during failover. If priorities are equal, instances are selected by
+    lexicographical order of their names.
+]])
+
+M['failover.replicasets.*'] = 'Name of the replicaset.'
```

Comment:
Since we're describing values of a map (not keys), I would say something like 'failover coordinator options related to a particular replicaset'.

Reply:
Ok. Let's do it this way.